pkg/monitoring/prometheusagent/types.go

```diff
@@ -13,8 +13,7 @@ const (
 	// servicePriorityLabel is the label used to determine the priority of a service.
 	servicePriorityLabel string = "giantswarm.io/service-priority"
 
-	remoteWriteEndpointTemplateURL = "https://prometheus.%s/%s/api/v1/write"
-	remoteWriteName                = "prometheus-meta-operator"
+	remoteWriteEndpointTemplateURL = "https://mimir.%s/api/v1/push"
```

Comment:
Why did you move the remoteWriteName over?

Reply:
That was a very early change, before knowing that the operator would be used only when mimir would be enabled. Reverting it :)